### PR TITLE
Minor PT-BR language string fixes.

### PR DESCRIPTION
**Pull request by d3fz**
Minor string fixes to some Portuguese-BR language files.

### CODE REVIEW

**Review by lantis1008 — CHANGES_REQUESTED** [review]
You appear to have removed the BOM from dhcp.js, quotas.js and restrictions.js. This will cause the pages to fail to load.

File: package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/dhcp.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿/*
```
Comment (lantis1008) [inline package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/dhcp.js:1]: BOM looks modified here

File: package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/quotas.js [hunk 2]
```diff
@@ -1,59 +1,59 @@
-﻿/*
```
Comment (lantis1008) [inline package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/quotas.js:1]: BOM looks modified here

File: package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/restrictions.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
-﻿/*
```
Comment (lantis1008) [inline package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/restrictions.js:1]: BOM looks modified here

**Comment by d3fz** [comment]
Thank you for pointing that out, I noticed that as well when i was updating these files.

Strange thing is, the whole line got automatically removed and than added right below , even though i didn't make any changes in this section.

`-/*
 +/*
`

When clicking to edit the mentioned files, it appears to be correct. 

Is this a bug or i am missing something ?

Any suggestions ?

**Pull request closed without merging**